util/package_test.go

```diff
@@ -126,3 +126,10 @@ func TestValidate(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkNewPackage(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		_, err := NewPackage("../testdata/package/reference/1.0.0")
```

Comment:
Can you also add a benchmark for NewPackageWithResources?

Reply:
Haha, interesting you ask. Because that is the one I plan to remove :-D And see what the difference in performance is.

Reply:
Please check then, if it doesn't break any validation or package_index handler.